com.unity.ml-agents/Tests/Editor/Inference/ParameterLoaderTest.cs

```diff
@@ -79,6 +79,8 @@ public BuiltInSensorType GetBuiltInSensorType()
     public class ParameterLoaderTest
     {
         const string k_discrete_ONNX_v2 = "Packages/com.unity.ml-agents/Tests/Editor/TestModels/discrete_rank2_vector_v2_0.onnx";
```

Comment:
Can we rename the old model file to something like "obsolete_lstm_v1.onnx"? And rename the constants and NNModel variables accordingly?

Reply:
I renamed to discrete1vis0vec_2_3action_obsolete_recurr_v1_0.onnx. I remade this model with version 15 since it seems this model was actually not a recurrent model.